change/@ni-nimble-components-078b375d-daec-46bc-8525-995725df1168.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Can you un-exclude them from renovate

https://github.com/ni/nimble/blob/d28658d75be9581b3910f047fc7ac1fdf4ebe6d4/.github/renovate.json#L30-L31